<launch>
    <node name="atom_imu" pkg="atom_imu" type="atom_imu" output="screen">
        <param name="pub_sync_time_topic" value="true" />
        <param name="pub_ros_time_topic" value="true" />
        <param name="port_name" value="/dev/ttyUSB1" />
        <param name="baud_rate" value="115200" />
    </node>
</launch>
